frontend/packages/ceph-storage-plugin/src/components/modals/advanced-kms-modal/advanced-vault-modal.tsx

```diff
@@ -28,8 +28,8 @@ export const AdvancedVaultModal = withHandlePromise((props: AdvancedKMSModalProp
 
   const { t } = useTranslation();
   const [backendPath, setBackendPath] = React.useState(kms?.backend || '');
-  const [authPath, setAuthPath] = React.useState(kms?.providerAuthPath || '');
-  const [authNamespace, setAuthNamespace] = React.useState(kms?.providerAuthNamespace || '');
+  // const [authPath, setAuthPath] = React.useState(kms?.providerAuthPath || '');
+  // const [authNamespace, setAuthNamespace] = React.useState(kms?.providerAuthNamespace || '');
```

Comment:
Let's just remove dead code. If it's required in future releases just revert the commit.

Reply:
ack